include/npcomp/Dialect/TCF/IR/TCFOps.td

```diff
@@ -95,4 +95,24 @@ def TCF_MatmulOp : TCF_Op<"matmul"> {
   let assemblyFormat = "$lhs `,` $rhs attr-dict `:` functional-type(operands, results)";
 }
 
+def TCF_ConvNCHWOp : TCF_Op<"conv_2d_nchw"> {
+  let summary = "2-D convolution";
+  let description = [{
+    Performs 2-D convolution. This op is inspired by PyTorch's Conv2d layer (https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html).
+
+    The tensors have dimensions:
+    - in:     [N, C, H, W]
+    - filter: [F, C, KH, KW]
+    - result: [N, F, Hout, Wout]
+
+    The tensors must met the following conditions; otherwise, this op aborts the program.
+    - H is greater than or equal to KH
```

Comment:
also mention that this op aborts the program if the above tensor dimensions don't match when we use the same symbol for two dimensions.